hmt_escrow/job.py

```diff
@@ -1432,9 +1436,10 @@ def _create_escrow(self,
                 escrow_created = True
                 break
             except Exception as e:
-                LOG.error(
-                    f"Contract creation failed with {gas_payer} and {gas_payer_priv} due to {e}. Raffling new ones..."
-                )
+                if gas_payer == self.multi_credentials[-1][0]:
+                    LOG.error(
```

Comment:
can you make this a `LOG.exception`?

Reply:
sure

Reply:
Hmm one other thing I was thinking here was - is there any action to take aside from the logging here? Ie do we cancel the contract? Or just not create it at all and throw exception?

Reply:
We'll retry it the next iteration if this time it fails as the status of the contract won't change

Reply:
Cancelling is probably too aggressive.